src/MainWindow.cpp

```diff
@@ -2423,6 +2428,19 @@ void MainWindow::onNewNoteCreationRequested()
     createNewNote(NoteEditorTabsAndWindowsCoordinator::NoteEditorMode::Any);
 }
 
+void MainWindow::onToggleThumbnailsPreference(QString noteLocalUid)
+{
+    QNDEBUG(QStringLiteral("MainWindow::onToggleThumbnailsPreference noteLocalUid=") << noteLocalUid);
+    bool toggleForAllNotes = noteLocalUid.isEmpty();
+    if (toggleForAllNotes) {
+        toggleShowNoteThumbnails();
+    } else {
```

Comment:
Please move this `else` statement onto the next line to comply with the coding style.